### PR TITLE
chore: simplify LightClient usage

### PR DESCRIPTION
**Pull request by jeluard**
**Motivation**

Introduce helper function to make it easier to use LightClient

### CODE REVIEW

File: packages/light-client/src/utils/utils.ts [hunk 1]
```diff
@@ -78,3 +80,19 @@ export function isEmptyHeader(header: BeaconBlockHeader): boolean {
...
+  ApiError.assert(res);
+
+  return {
+    genesisTime: res.response.data.genesisTime,
+    genesisValidatorsRoot: res.response.data.genesisValidatorsRoot,
+  };
+}
+
+export async function getFinalizedSyncCheckpoint(api: Pick<Api, "beacon">): Promise<Bytes32> {
+  const res = await api.beacon.getStateFinalityCheckpoints("head");
```
Comment (nazarhussain) [inline packages/light-client/src/utils/utils.ts:95]: Should we add a check here to make sure it returns the finalized head. 

As per the spec the endpoint can return zero hash if finality not achieved.

**Review by nazarhussain — CHANGES_REQUESTED** [review]

File: packages/light-client/src/utils/utils.ts (hunk 1 above)
Comment (jeluard) [reply packages/light-client/src/utils/utils.ts:95]: I would then expect that the `root` is an empty `Bytes32` (i.e. `Uint8Array`)?
Alternatively, we can return null in this case.

Comment (nazarhussain) [reply packages/light-client/src/utils/utils.ts:95]: That's totally an edge case for testing, when you have a custom new network. So I would skip this point for now.

**Review by nazarhussain — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.17.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.17.0) :tada:

**Pull request merged**